lib/jerboa/client.ex

```diff
@@ -121,6 +121,22 @@ defmodule Jerboa.Client do
   end
 
   @doc """
+  Creates permission on the allocation for the given peer
+  address
+
+  If permission is already installed for the given address,
+  the permission will be refreshed.
+
+  ## Example
+
+      iex> create_permission client, {192, 168, 22, 111}
+  """
+  @spec create_permission(t, peer :: ip) :: :ok | {:error, error}
```

Comment:
Oh, I see... Well, server assumes that it can iterate though params to match and fetch multiple peers. And the standard says, that it `MUST` be possible. Therefore I think that we need to at least add task for extending encoder/decoder to our backlog. I don't mind that `Params.get_attr/2` returns first occurrence of attribute. That's fine. The most important thing is that decoder has to return multiple peer attributes and doesn't skip anything. 

If the client part - encoding - is too much work, then we may skip it for now, but we should have a ticket for that IMHO.